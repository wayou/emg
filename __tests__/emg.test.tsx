import * as React from "react";
import { create } from "react-test-renderer";
import Emg from "../src/emg";
import ExampleImg from "./images/example.png";

test("image should rendered", () => {
  const component = create(<Emg src={ExampleImg} />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
