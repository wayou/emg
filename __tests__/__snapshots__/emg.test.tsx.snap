// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`image should rendered 1`] = `
<img
  alt=""
  className="image  image-loading"
/>
`;
